libnmstate/nm/checkpoint.py

```diff
@@ -43,6 +46,12 @@ class NMCheckPointCreationError(NMCheckPointError):
     pass
 
 
+class NMCheckPointPermissionError(NMCheckPointError):
+    """ Error creating a Network Manager CheckPoint """
```

Comment:
I think the name of the error is informative enough, this extra doctext adds no informative info.

Reply:
fixed, also adjusted the other docstrings